conf/application.conf

```diff
@@ -65,6 +65,17 @@ play.modules.enabled +="uk.gov.hmrc.play.bootstrap.AuditModule"
 
 play.http.errorHandler = "uk.gov.hmrc.teamsandrepositories.errorhandlers.LoggingErrorHandler"
 
+play.http.filters="uk.gov.hmrc.teamsandrepositories.Filters"
+play.filters.cors {
+  # allow all paths
+  pathPrefixes = ["/"]
+  # allow all origins (You can specify if you want)
+  allowedOrigins = null
```

Comment:
Probably not a big deal but worth looking at this https://w3c.github.io/webappsec-cors-for-developers/#avoid-returning-access-control-allow-origin-null